rn/Teacher/src/common/components/rows/RowWithTextInput.js

```diff
@@ -58,7 +58,7 @@ export default class RowWithTextInput extends Component<Props, any> {
     const label = i18n('{value}, text field', {
       value: this.props.value || this.props.placeholder || i18n('Input'),
     })
-    return <Row {...this.props} children={this._renderTextInput()} accessibilityLabel={label} />
+    return <Row {...this.props} children={this._renderTextInput()} accessibilityLabel={label} accessible={false} />
```

Comment:
Since this view contains a text field, the row can't be considered as one text field or else XCUITest can't type into it.

Reply:
Sounds good. I assume this will break some snapshots though. You may need to run `yarn test -u` and commit the updated snapshots.